colossalai/zero/sharded_optim/sharded_optim_v2.py

```diff
@@ -142,8 +182,13 @@ def step(self, *args, **kwargs):
                 # Now p.data is sharded
                 # So optimizer states are sharded naturally
 
+        self._logger.info(f"Before step ShardedOptimizerV2 consumes {self.get_memory_usage()[0]/1e6} MB CUDA Memory!",
```

Comment:
Can we use `logger.debug` in this class?

Reply:
Sure